internal/internal_workflow_client.go

```diff
@@ -263,6 +263,89 @@ func (wc *workflowClient) SignalWorkflow(ctx context.Context, workflowID string,
 		}, serviceOperationRetryPolicy, isServiceTransientError)
 }
 
+// SignalWithStartWorkflow sends a signals to a workflow in execution, if workflow is not running or not found,
```

Comment:
suggested update:
SignalWithStartWorkflow sends a signal to a running workflow. If the workflow is not running or not found, it starts the workflow and then sends the signal in transaction.